README.md

```diff
@@ -96,6 +96,7 @@ Help Options:
 ## Documentation
 
 When ready we will put up a full documentation of the OpenBazaar protocol and the JSON API at https://docs.openbazaar.org
+Currently, the docs can be accessed from https://api.openbazaar.org/
```

Comment:
docs.openbazaar.org does not appear active at the moment. Let's replace the whole section with the following:
```
Documentation of the OpenBazaar protocol has not been formalized yet. If you would like to help, please reach out in [slack](https://openbazaar.slack.com/) or via a new issue on github.

`openbazaar-go` exposes an HTTP API which permits high-level interactions on the network and the internal wallet. Find the HTTP API documentation at [https://api.openbazaar.org](https://api.openbazaar.org).
```